lib/Service/SettingsService.php

```diff
@@ -240,6 +243,23 @@ public function setMinimumConfidence($confidence) {
 	 * The next settings are advanced preferences that are not available in gui.
 	 * See: https://github.com/matiasdelellis/facerecognition/wiki/Settings#hidden-settings
 	 */
+	private function getModelPath(): string {
+		// Get model path setting
```

Comment:
NO!.
here just return the value as before.. The differentiation between both folders must be done in the Model Service!